components/ArticlesList.jsx

```diff
@@ -0,0 +1,43 @@
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { ArticleCard } from "./ArticlesCard";
+
+export const ArticlesList = () => {
+    const [articles, setArticles] = useState([]);
+    const [isLoading, setIsLoading] = useState(true);
+    const [isError, setIsError] = useState(false);
+
+    useEffect(() => {
+        axios
+            .get(`https://nc-news-lb25.onrender.com/api/articles`)
+            .then((response) => {
+                setArticles(response.data.articles);
+                setIsLoading(false);
+            })
+            .catch((error) => {
+                setIsLoading(false);
+                setIsError(true);
+            })
+    }, []);
+
+    if (isError) return <p>Something went wrong!</p>
+    if (isLoading) return (
+        <div className="d-flex justify-content-center">
```

Comment:
It might be goof to separate this loading component into a component to be used in other places, this is not essential, just a consideration.